### DISCUSSION TITLE
Remove Edit Icon when can_edit is False

### DISCUSSION THREAD

**Discussion by sternalon** [jowilf/starlette-admin#320] [General]
How can I remove the edit icon from the list page for users who do not have edit permissions.

Currently, users without edit permission click on the edit icon and are then given a 403 error. It would be preferable not to show them the edit icon at all.

Thanks
Alon

**Comment by hasansezertasan** [comment]
> How can I remove the edit icon from the list page for users who do not have edit permissions.
> 
> Currently, users without edit permission click on the edit icon and are then given a 403 error. It would be preferable not to show them the edit icon at all.
> 
> Thanks Alon

Is this still happening?
